pylint/plugins/hass_enforce_type_hints.py

```diff
@@ -717,6 +718,34 @@ class ClassTypeHintMatch:
             ],
         ),
     ],
+    "calendar": [
+        ClassTypeHintMatch(
+            base_class="Entity",
+            matches=_ENTITY_MATCH,
+        ),
+        ClassTypeHintMatch(
+            base_class="CalendarEntity",
+            matches=[
+                TypeHintMatch(
+                    function_name="event",
+                    return_type=["CalendarEvent", None],
+                ),
+                TypeHintMatch(
+                    function_name="state",
+                    return_type=["str", None],
+                ),
```

Comment:
Hmm... I don't think state should be implemented on the integration platform level.
(I wonder why its not marked as final)

Reply:
I have adjusted this to mark it as final, and drop it from the pylint plugin